packages/svelte-ux/src/lib/components/NavItem.svelte

```diff
@@ -16,7 +16,10 @@
   export let classes: {
     root?: string;
     active?: string;
-    icon?: string;
+    icon?: string | {
+      root?: string |;
```

Comment:
Bad `;` character.  We can also change this to `icon?: ComponentProps<Icon>['classes']` (ex. [Card](https://github.com/techniq/svelte-ux/blob/main/packages/svelte-ux/src/lib/components/Card.svelte#L18)

I'm OK with making this a breaking change... I don't know if anyone would be using `<NavItem classes={{ icon: '' }}>` and I think breaking now instead of continuing to support the use case is best.